opentech/apply/activity/messaging.py

```diff
@@ -388,8 +388,15 @@ def extra_kwargs(self, message_type, **kwargs):
             'links': links,
         }
 
-    def recipients(self, message_type, submission, **kwargs):
-        return [self.slack_id(submission.lead)]
+    def recipients(self, message_type, submission, related, **kwargs):
+        recipients = [self.slack_id(submission.lead)]
+
+        # Notify second reviewer when first reviewer is done.
+        if message_type == MESSAGES.NEW_REVIEW:
```

Comment:
```suggestion
        if message_type == MESSAGES.NEW_REVIEW and related:
```

We can have check for related to avoid AttributeError if related is None.